abhi-script.py

```
@@ -0,0 +1,66 @@
+from typing import List
+import numpy as np
+from transformers import AutoTokenizer
+import random
+from tqdm import tqdm
+from datasets import load_dataset
+
+DATA_DIR = "/new_data/wenlong/wz-entigraph-data/data/dataset/raw"
+BIN_DIR = "/new_data/wenlong/wz-entigraph-data/data/dataset/bins"
+
+# /new_data/knowledge_rh/quality/training_mix/exp-2-x/level1_level2_level3_v2_test_set_qa3_d_synth_percent_154%.jsonl
+def _get_quality_graph(model_name: str) -> List[str]:
```

Ruff (0.14.5)

Unused function argument: `model_name`

(ARG001)